src/index.html

```diff
@@ -10,23 +10,51 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Document</title>
+    <title>Search bar</title>
     <link
       rel="stylesheet"
       href="style.css"
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <div class="search__container">
+      <form
+        data-qa="big"
+        class="search__form search__form--big"
+      >
+        <span class="search__icon-container search__icon-container--big">
+          <img
+            src="/src/images/Search.svg"
```

Comment:
Ensure that the path '/src/images/Search.svg' is correct and that the image file exists at this location. If the image is not displaying, this might be the cause.